xmodule/seq_block.py

```diff
@@ -978,8 +978,43 @@ def definition_to_xml(self, resource_fs):
         xml_object = etree.Element('sequential')
         for child in self.get_children():
             self.runtime.add_block_as_child_node(child, xml_object)
+        self.add_prerequisite_to_xml(xml_object)
         return xml_object
 
+    def add_prerequisite_to_xml(self, xml_object):
+        """
+        Add prerequisite information to sequential XML for export.
+        """
+        from openedx.core.lib.gating import api as gating_api
+
+        prereq_info = gating_api.get_required_content(
+            self.location.course_key,
+            self.location
+        )
+        prereq_id, min_score, min_completion = prereq_info
```

Comment:
```suggestion
        prereq_usage_key_str, min_score, min_completion = prereq_info
```

The `id` is unclear here.